scrapy/spiders/sitemap.py

```diff
@@ -31,6 +31,14 @@ def start_requests(self):
         for url in self.sitemap_urls:
             yield Request(url, self._parse_sitemap)
 
+    def sitemap_filter(self, urls):
```

Comment:
I think `urls` name is confusing, as one would expect it is a list of strings, not a list of dicts with information. Any ideas about a better name for this argument? We may also need to document it better, at least to explain which keys are available.

Reply:
Sure. I had the same thought but weren't able to think about a better name. Indeed they are <url> tags being iterated... As in the .xml file.

Reply:
what about `links` or `snippets` ?

Reply:
`links` will lead to the same confusion, someone would expect a list of strings... 

I like `entries` tho.

Reply:
+1 to entries, for links I'd expect a list of scrapy.link.Link instances.